src/test/resources/features/04_employment.feature

```diff
@@ -209,4 +210,74 @@ Feature: Employment Journeys - Income Tax Submission
+    When the user selects the PAYE reference field and enters a value of 777/7777
+    And the user clicks the continue button
+    Then the user should be redirected to the "Check your client’s employment details" page
+    When the user clicks on the change pay received link
+    Then the user should be redirected to the "How much did your client’s employer pay them?" page
+    When the user selects the Pay field and enters a value of 9000.01
+    And the user clicks the continue button
+    Then the user should be redirected to the "Check your client’s employment details" page
+    When the user clicks on the change tax taken from pay link
+    Then the user should be redirected to the "How much UK tax was taken from your client’s earnings?" page
+    When the user selects the Tax field and enters a value of 720.08
+    And the user clicks the continue button
+    Then the user should be redirected to the "Check your client’s employment details" page
+
+    Scenario: Agent user with at least one prior employment removes an employment
+        When the user logs into the service with the following details
+          |Redirect url       |/test-only/2021/additional-parameters?ClientNino=BB444444A&ClientMTDID=1234567890  |
+          |Credential Strength|weak                                                                               |
+          |Confidence Level   |200                                                                                |
+          |Affinity Group     |Agent                                                                              |
+          |Enrolment Key 1    |HMRC-MTD-IT                                                                        |
+          |Identifier Name 1  |MTDITID                                                                            |
+          |Identifier Value 1 |1234567890                                                                         |
+          |Enrolment Key 2    |HMRC-AS-AGENT                                                                      |
+          |Identifier Name 2  |AgentReferenceNumber                                                               |
+          |Identifier Value 2 |XARN1234567                                                                        |
+        Then the user should be redirected to the "Update and submit an Income Tax Return" page
+        When the user clicks the continue button
+        Then the user should be redirected to the "Your client’s Income Tax Return" page
+        When the user clicks on the employment link
+        Then the user should be redirected to the "Employment" page
+        When the user clicks on the third remove link
+        Then the user should be redirected to the "Are you sure you want to remove this employment?" page
+        When the user selects the yes radio button
+        And the user clicks the continue button
```

Comment:
same here - should this flow include the next page after continue button (line 282)?